usvg/src/image_href_resolver/default_resolver.rs

```diff
@@ -0,0 +1,140 @@
+//! Default resolver that `usvg` uses to handle `<image>` elements `xlink:href` strings.
```

Comment:
I don't think we need a separate `image_href_resolver` module. All this code can leave in `image.rs` just fine.